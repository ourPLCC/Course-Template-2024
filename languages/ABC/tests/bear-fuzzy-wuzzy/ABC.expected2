

2 matches:
bear(fuzzy)
bear(wuzzy)

