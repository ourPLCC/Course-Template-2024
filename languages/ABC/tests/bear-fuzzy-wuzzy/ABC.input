bear(fuzzy).
bear(wuzzy).
bear(X)?    % yields "bear(fuzzy)" and "bear(wuzzy)"