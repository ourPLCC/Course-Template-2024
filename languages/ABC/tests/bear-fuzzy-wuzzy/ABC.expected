

2 matches:
bear(wuzzy)
bear(fuzzy)

